### ISSUE TITLE
Train custom data, batch & subdivisions

### ISSUE THREAD

**Issue by LFRS** [pjreddie/darknet#2398]
Hi,

On training a custom dataset do we have any advantage in increasing the value of "batch" and decrease the value of "subdivisions"?

For example:
batch=64 -> batch=256
subdivisions=32 -> subdivisions=1

What behavior can we expect to change these parameters? And what implications does it have on loss and avg loss?

Thanks,
Luis

**Comment by alexchoihku** [comment]
In case I have 60 classes, and ~3300 images training dataset, then batch size should be?
I'm confused since there have no papers mentioned. But about the subdivision, I think it is more related to the computer performance.
